scripts/tcs_listener.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+listener="${TCF_HOME}/examples/common/python/connectors/direct/tcs_listener/tcs_listener.py"
+# Read Listener port from config file
+listener_port=`grep listener_port ${TCF_HOME}/config/tcs_config.toml | awk {'print $3'}`
+
+# config in env variables takes higher priority
+port=${TCF_TCS_LISTENER_PORT:-${listener_port}}
```

Comment:
Is it TCS_LISTENER_PORT or TCF_TCS_LISTENER_PORT ? docker/k8s/tcf-config-map.yaml sets TCS_LISTENER_PORT

Reply:
It's `TCF_TCS_LISTENER_PORT ` since the `prefix` key is set `TCF_` in `envFrom` of all resources manifests (all files with suffix `.yaml`).